yarn.lock

```diff
@@ -7848,15 +7876,19 @@ __metadata:
     eslint: ^7.4.0
     eslint-config-prettier: ^6.11.0
     eslint-plugin-prettier: ^3.1.4
+    file-loader: ^6.2.0
     fs-extra: ^7.0.0
     identity-obj-proxy: ^3.0.0
     jest: ^29.7.0
     jest-cli: ^29.7.0
     jest-environment-jsdom: ^29.5.0
+    jest-transform-stub: ^2.0.0
     jsdom-global: ^3.0.2
+    mini-svg-data-uri: ^1.4.4
     mkdirp: ^0.5.1
     npm-run-all: ^4.1.3
     prettier: ^2.0.5
+    raw-loader: ^4.0.2
```

Comment:
If I don't include raw-loader, I get the following compilation error. We aren't specifically using raw-loader, but it seems like jupyter notebook is somewhere / somehow?

```
Error: resolve 'raw-loader' in '/Users/alli.scibisz/nbsv'
  Parsed request is a module
  using description file: /Users/alli.scibisz/nbsv/package.json (relative path: .)
    resolve as module
      looking for modules in /Users/alli.scibisz/nbsv/node_modules
        single file module
          using description file: /Users/alli.scibisz/nbsv/package.json (relative path: ./node_modules/raw-loader)
            no extension
              /Users/alli.scibisz/nbsv/node_modules/raw-loader doesn't exist
            .js
              /Users/alli.scibisz/nbsv/node_modules/raw-loader.js doesn't exist
        /Users/alli.scibisz/nbsv/node_modules/raw-loader doesn't exist
      /Users/alli.scibisz/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
    at compilerCallback (/Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/lib/build-labextension.js:100:19)
    at /Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/lib/build-labextension.js:142:21
    at /Users/alli.scibisz/nbsv/node_modules/neo-async/async.js:2830:7
    at done (/Users/alli.scibisz/nbsv/node_modules/neo-async/async.js:2865:11)
    at /Users/alli.scibisz/nbsv/node_modules/neo-async/async.js:2818:7
    at /Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/node_modules/webpack/lib/HookWebpackError.js:68:3
    at Hook.eval [as callAsync] (eval at create (/Users/alli.scibisz/nbsv/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/Users/alli.scibisz/nbsv/node_modules/tapable/lib/Hook.js:18:14)
    at Cache.shutdown (/Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/node_modules/webpack/lib/Cache.js:150:23)
    at /Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/node_modules/webpack/lib/Compiler.js:1229:15
```